src/main/java/thePackmaster/cards/Rummage.java

```diff
@@ -29,7 +29,7 @@ public void update() {
                 ArrayList<AbstractCard> possCardsList = new ArrayList<>();
 
                 for (AbstractCard q : AbstractDungeon.player.hand.group) {
-                    if (!q.freeToPlay())
+                    if (!q.freeToPlay() && q.cost>=0)
```

Comment:
```suggestion
                    if (!q.freeToPlay() && q.cost>0)
```
You don#t want to decrease the cost of a 0 cost card